integration-cli/docker_cli_kill_test.go

```diff
@@ -40,6 +40,28 @@ func (s *DockerSuite) TestKillContainer(c *check.C) {
 
 }
 
+func (s *DockerSuite) TestKillofStoppedContainer(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "top")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal(out, err)
+	}
+
+	cleanedContainerID := strings.TrimSpace(out)
+
+	stopCmd := exec.Command(dockerBinary, "stop", cleanedContainerID)
+	if out, _, err = runCommandWithOutput(stopCmd); err != nil {
+		c.Fatalf("failed to stop container: %s, %v", out, err)
+	}
+
+	killCmd := exec.Command(dockerBinary, "kill", "-s", "30", cleanedContainerID)
+	if _, _, err = runCommandWithOutput(killCmd); err == nil {
```

Comment:
I would really prefer if we checked the error message too so that we know your code is being hit instead of something else going wrong (like sig 30 isn't supported).